modules/disabling-insights-operator-gather.adoc

```diff
@@ -33,45 +35,47 @@ endif::openshift-rosa,openshift-dedicated[]
 . On the *CustomResourceDefinition details* page, click the *Instances* tab.
 . Click *cluster*, and then click the *YAML* tab.
 . Disable the gather operations by performing one of the following edits to the `InsightsDataGather` configuration file:
-.. To disable all the gather operations, enter `all` under the `disabledGatherers` key:
+.. To disable all the gather operations and data collection, define the `gatherers` specification and set the `mode` to *None* as outlined in the following example extract:
 +
 [source,yaml]
 ----
-apiVersion: config.openshift.io/v1alpha1
-kind: InsightsDataGather
-metadata:
-....
 
-spec: <1>
-  gatherConfig:
-    disabledGatherers:
-      - all <2>
+apiVersion: insights.openshift.io/v1alpha2
+kind: DataGather
```

Comment:
I believe this one is about periodic gathering, so this should be `kind: InsightsDatGather` and not `DataGather`